flink-table-store-core/src/main/java/org/apache/flink/table/store/CoreOptions.java

```diff
@@ -527,6 +535,17 @@ public double orcBloomFilterFpp() {
         return options.get(ORC_BLOOM_FILTER_FPP);
     }
 
+    public Map<Integer, String> fileCompressionPerLevel() {
+        Map<String, String> levelCompressions = options.get(FILE_COMPRESSION_PER_LEVEL);
+        if (null != levelCompressions) {
+            return levelCompressions.entrySet().stream()
+                    .collect(
+                            Collectors.toMap(
+                                    e -> Integer.valueOf(e.getKey()), Map.Entry::getValue));
+        }
+        return null;
```

Comment:
Maybe we can return an empty `Map` to avoid `if (fileCompressionPerLevel != null)`?